ethereum/contracts/Campaign.sol

```diff
@@ -29,21 +30,21 @@ contract Campaign {
     function contribute() public payable {
         require(msg.value > minimumContribution);
 
-        approvers[msg.sender] = true;
-        approversCount++;
+        if (approvers[msg.sender] != true) {
+          approvers[msg.sender] = true;
+          approversCount++;
+        }
```

Comment:
This fixes the bug that caused the approvers count to go up even when the person has already donated.